GitUI/Plugin/GitPluginSettingsContainer.cs

```diff
@@ -35,5 +36,15 @@ public override void SetValue<T>(string name, T value, Func<T, string> encode)
         {
             ExternalSettings.SetValue(_pluginName + name, value, encode);
         }
+
+        public override void SetCredential(string name, NetworkCredential value, IGitModule gitModule)
```

Comment:
```suggestion
        public override void SetCredentials(string name, NetworkCredential value, IGitModule gitModule)
```

Reply:
It is not correct, because method take one credential, but not set of credentials